site/src/playground/export/lib.less

```diff
@@ -1,5 +1,4 @@
 @import '../../common/reboot.less';
-@import '../../common/typography.less';
```

Comment:
This will break the styles
I think we should use our typography in our examples.
If I understand correctly, that change relates to the default colors or padding, then I'd recommend 
  - for colors -> move them to the propper place (e.g. to `colors.less` instead of typography)
  - for padding -> just override them or move to `page.less` if this kind of changes required at all